biojava-structure/src/main/java/org/biojava/nbio/structure/SVDSuperimposer.java

```diff
@@ -131,11 +131,11 @@
  */
 public class SVDSuperimposer {
```

Comment:
This would be a good time (since we are releasing a new major version) to move this class to the geometry package too. Any reasons not to do it anyone?